test/native/snapshot/pp.expected

```diff
@@ -2701,6 +2701,898 @@ module Component = {
 
 Css.style([Css.unsafe("display", "block")]);
 
+module Component = {
+  [@bs.deriving abstract]
+  type makeProps('var) = {
```

Comment:
makeProps now will have a type parameter for each component parameter

BTW, snapshots are still useful for seeing the difference in the generated code